### ISSUE TITLE
Django 2.1 permalink

### ISSUE THREAD

**Issue by axeliodiaz** [arneb/django-messages#117]
[Django 2.1](https://docs.djangoproject.com/es/2.1/releases/2.1/) has remove decorator 'permalink' and django-messages crash:

```
.venv/src/django-messages/django_messages/models.py", line 82, in Message
    get_absolute_url = models.permalink(get_absolute_url)
AttributeError: module 'django.db.models' has no attribute 'permalink'
```

**Comment by axeliodiaz** [comment]
Is maybe fix with this [pull request](https://github.com/arneb/django-messages/pull/119/commits/018592396664e9f64298cfd2b14bbebaeb72dba1) (¿?)

**Comment by DChristopher** [comment]
Hi Axelio, sorry I'm a bit rusty with GitHub. I am trying to use Django_messages with Django 2.1.3. Is there a version that contains your fix? or a way I could get it locally for now?

**Comment by 3skr0** [comment]
up on this issue ?

**Comment by arneb** [comment]
fixed in 0.6.0
